.travis.yml

```diff
@@ -22,6 +22,10 @@ script:
 - travis-cargo test
 - travis-cargo --only stable test -- --features=deadlock_detection
 - travis-cargo --only beta test -- --features=deadlock_detection
+- if [ "${TRAVIS_RUST_VERSION}" = "nightly" ]; then
+    rustup target add x86_64-fortanix-unknown-sgx --toolchain nightly;
+  fi
+- travis-cargo --only nightly test --target x86_64-fortanix-unknown-sgx
```

Comment:
You will not be able to run this on Travis. But you can do `--no-run`.

Reply:
Oh, ok :/ Doing that for now.